status/running_notes.py

```diff
@@ -374,6 +374,32 @@ def get(self, partitionid):
             self.write({latest_sticky_note["created_at_utc"]: latest_sticky_note})
 
 
+class LatestStickyNotesMultipleHandler(SafeHandler):
+    """Serves the latest sticky running note for multiple projects.
+
+    URL: /api/v1/latest_sticky_run_note)
+    """
+
+    def post(self):
```

Comment:
If we are only fetching data, this should be `get`

Reply:
While semantically I agree with you, I'm not sure it's correct. I think get has much more limitations than post. I believe that a get request should ideally not be dependant on a request body. Putting all the project_ids into the URI is a bit too clunky...

Reply:
Yes, that would be clunky